### DISCUSSION TITLE
Builds (0H223, 0H224 et cetera)

### DISCUSSION THREAD

**Discussion by grahamperrin** [helloSystem/hello#410] [Q&A] [answered]
At <https://github.com/helloSystem/ISO/releases/tag/experimental-13.1>, what was the difference between `hello-0.8.0_0H223-FreeBSD-13.1-amd64.iso` and `hello-0.8.0_0H224-FreeBSD-13.1-amd64.iso`?

I can no longer find the issues where build naming was discussed, neither can I find an explanation at <https://hellosystem.github.io/>.

**Comment by probonopd** [comment]
Build Train:
* 0 = the "0" in 0.8.0
* H = the 8th letter in the alphabet, for the "8" in 0.8.0

Build Number:

* 223, 224,... the 223th, 224th build toward what will become 0.8.0 one it gets released. There is no dedicated changelog for each build but changes worthy to be mentioned in the 0.8.0 changelog are collected at https://github.com/helloSystem/hello/wiki/Status/#pre-080

Mac users will find this familiar: https://tidbits.com/2020/07/08/how-to-decode-apple-version-and-build-numbers/

* https://github.com/helloSystem/buildnumbers

**Reply by grahamperrin** [reply]
Thanks, can you find the naming-related issues in GitHub?

**Reply by probonopd** [reply]
Hello @grahamperrin, what do you mean by "the naming-related issues"? Issues regarding the buildnumbers system can be placed here: https://github.com/helloSystem/buildnumbers/issues

**Reply by grahamperrin** [reply]
There were discussions that took the form of GitHub issues or PRs. 

If I recall correctly: 

* issues, not PRs
* at least one in the helloSystem organisation
* one, maybe two, in <https://github.com/bsdhw/> or <https://github.com/linuxhw/>.

**Comment by grahamperrin** [comment, marked as answer]
Found: 

* https://github.com/helloSystem/hello/issues/67

----

From <https://github.com/helloSystem/hello/discussions/410#discussioncomment-4246076>: 

> * one, maybe two, in [@bsdhw](https://github.com/bsdhw/) or [@linuxhw](https://github.com/linuxhw/).

Now, I realise why I couldn't find things there. The issue above was begun by @linuxhw in the helloSystem organisation (not in a probe-related repo).
